MAUI/DataForm/Data-annotations.md

```diff
@@ -0,0 +1,376 @@
+---
+layout: post
+title: Data annotations in .NET MAUI DataForm control | Syncfusion
+description: Learn here all about Data annotations support in Syncfusion Maui DataForm (SfDataForm) control to handle the data.
```

Comment:
Learn about the Data annotations support in Syncfusion Maui DataForm (SfDataForm) control in mobile and desktop applications from a single shared codebase.

Reply:
description added